source/whats-new.txt

```diff
@@ -41,6 +41,11 @@ What's New in 1.11
 
 - Added the ``UpdateOneDefaultStrategy`` write model strategy.
 
+- Added the ``change.stream.document.key.as.key`` source connector
+  configuration property so tombstone events use the keys of deleted
+  documents as source record keys. To learn more about this property,
+  see the list of :ref:`Change Stream Properties <source-configuration-change-stream-list>`.
```

Comment:
Issue:
From Ross's comment, this could be a breaking change since it defaults to true and to revert the behavior, set it to false. I think it could be helpful to mention that here.